tn_api_spec.yml

```diff
@@ -1180,6 +1180,61 @@ components:
             type: string
             description: Fare Info reference for this segment
             example: 3RxQ6ylJ0BKAhQAbnbAAAA==
+          virtual_interline:
+            type: boolean
+            description: |
+              true if option is a vi solution
+            example: true
+          vi_solution_id:
+            type: string
+            description: |
+              ID mapping VI parts of the option together. Helps finding VI solutions located in same segment position. 
+              Will be composed of each segment's segment_id joined together with two underscores. For oneway VI this 
+              will be two segments, for openjaw VI will be 4 segments. 
+            example: fc906f39e5151f1bc5ef788b85f3d280770b5b7c__b9ebd56ff7bf07a65a3e4dc70e9551e4ae50f1da
+            nullable: true
+          itinerary_index:
+            type: int
+            description: |
+              Position of the virtual interlining option in itinerary(PNR). Starts from 0
+              Helps ordering parts of the VI solution in relation to the itinerary as a whole
+            example: 0
+            nullable: true
+          vi_position:
+            type: integer
+            description: |
+              Position of the virtual interlining option in the VI itinerary. Starts from 0. For OneWay VI will be either of [0, 1]. 
+              For OpenJaw VI will be one of [0, 1, 2, 3]
+            example: 1
+            nullable: true
+          itinerary_structure:
+            type:  String
+            description: |
+              "String Array representing the structure of the trip in terms of one-ways and open-jaw segments. 
```

Comment:
plz remove double quotes as it looks like this in the docs 
![image](https://user-images.githubusercontent.com/99681496/234326871-16bd6339-068e-40e3-8b35-bdf8c84e26e1.png)